src/core/util/git_tracker.cc

```diff
@@ -15,14 +15,24 @@
 #ifdef USE_LIBGIT2
 
 #include "git_tracker.h"
-#include <experimental/filesystem>
 #include "core/util/log.h"
 #include "fstream"
 #include "git2.h"
 #include "iostream"
 #include "stdio.h"
 
+#ifdef __APPLE__
```

Comment:
Can we avoid code duplication by replacing all this with `#include "core/stdfilesystem.h"`?
This was introduced in b732247. The git_tracker.cc file was probably overlooked because it is an optional build target that is turned off by default.

Reply:
Sure, thank you for pointing that out.